### PR TITLE
Update storage & delete unnecessary files

### PR DESCRIPTION
**Pull request by A1WAYSD**

### CODE REVIEW

File: src/main/java/seedu/flashlingo/storage/JsonAdaptedFlashCard.java [hunk 1]
```diff
@@ -48,12 +47,18 @@ public JsonAdaptedFlashCard(@JsonProperty("originalWord") String originalWord,
...
     public JsonAdaptedFlashCard(FlashCard source) {
-        originalWord = source.getOriginalWord().toString();
+        originalWord = source.getOriginalWord().getWord();
         originalWordLanguage = source.getOriginalWord().getLanguage();
-        translatedWord = source.getTranslatedWord().toString();
+        translatedWord = source.getTranslatedWord().getWord();
         translatedWordLanguage = source.getTranslatedWord().getLanguage();
-        whenToReview = source.getWhenToReview().toString();
         level = source.getProficiencyLevel().getLevel();
+        //TODO: getLevel redundant?
```
Comment (itsNatTan) [inline src/main/java/seedu/flashlingo/storage/JsonAdaptedFlashCard.java:55]: Why do you feel getLevel redundant?

**Review by itsNatTan — APPROVED** [review]
LGTM, please get back to me on why you feel getLevel is redundant

File: src/main/java/seedu/flashlingo/storage/JsonAdaptedFlashCard.java (hunk 1 above)
Comment (A1WAYSD) [reply src/main/java/seedu/flashlingo/storage/JsonAdaptedFlashCard.java]: Nope. Sorry for my confusion about method names previously. Already corrected in new commits.

Comment (itsNatTan) [reply src/main/java/seedu/flashlingo/storage/JsonAdaptedFlashCard.java]: 👍

**Pull request merged**